client/src/index.jsx

```diff
@@ -1,21 +1,86 @@
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
+/* eslint-disable no-console */
+
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import styled, { injectGlobal } from 'styled-components';
+import axios from 'axios';
+import ModOne from './pricing_mod1/ModOne';
+// import ModTwo from "./calendar_mod2/ModTwo";
+// import ModThree from './guestsAndCalc_mod3/ModThree';
+
+injectGlobal( [ `
+  html, body{
+    height: 100%;
+    width: 100%
+  }
+  body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+` ] );
+
+export const Holder = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 300px;  
+  height: 300px;
+  border: 1px solid rgb(172, 172, 172);
+  font-family: Quicksand;
+`;
 
 export default class App extends Component {
   constructor( props ) {
     super( props );
+    const id = this.getPathname();
     this.state = {
-      here: 1,
+      pathname: id,
     };
   }
 
+  componentDidMount() {
+    this.getPathname();
+    this.getListingData( this.state.pathname );
+  }
+
+  getPathname = () => {
+    const id = window.location.pathname;
+    if ( id === '/' ) {
+      return 1;
+    }
+    return id.substr( 1 );
+  }
+
+  getListingData = ( id ) => {
+    let idToUse = id;
```

Comment:
You shouldn't need to create another variable here because you're not modifying id.

Reply:
The linter was yelling at me :(